src/configManager/helpers/configDataMap.js

```diff
@@ -29,7 +29,11 @@ let configDataMap = {
             rejectUnauthCerts: process.env.SMTP_REJECT_UNAUTH_CERTS || false
         }),
         type: 'json'
-    }
+    },
+    [constConfig.THRESHOLD]: { value: process.env.THRESHOLD, type: 'int' },
```

Comment:
not sure i get this

as I see it there is only BENCHMARK_CONFIG which saves everything under it (currently)

I would remove BENCHMARK_CONFIG
and keep 3 different params 
THRESHOLD -> BENCHMARK_THRESHOLD
THRESHOLD_WEBHOOK_URL -> BENCHMARK_THRESHOLD_WEBHOOK_URL
BENCHMARK_WEIGHTS -> BENCHMARK_WEIGHTS

this will allow us send each of them without overriding the other
you might need to change the swagger to support it (not sure, depends how you do it)